website/docs/reference/node-selection/configure-state.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Configure state selection"
+description: "Learn how to configure state selection in dbt."
```

Comment:
For SEO I think we should include defer here and in the title
```suggestion
description: "Learn how to configure state selection and set defer in dbt."
```

Reply:
On second thought - Defer already exists in this same section. Maybe we should move defer down to where this article is and link out to it instead